recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/xendriverdomain.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
Why is that we need to craft this file ourselves now? how was it implemented w/o this file before thud?

Reply:
Now it is provided by meta-virtualization layer. So, removing it from guest-addons and let Xen install it.